### PR TITLE
Add Turkey Ministry Of Interior Terrorist Wanted List

### PR DESCRIPTION
**Pull request by fjuniorr**
Closes https://github.com/opensanctions/crawler-planning/issues/108

@jbothma a couple of questions:

1) I'm getting a `assert self.name == slugify(self.name, sep="_"), "Dataset name is invalid"` when crawling on my end but not sure where it's coming from. Could you check if you are getting the warning as well? 

2) The birth date in the dataset has year granularity. I found only one mention of this in the [codebase](https://github.com/opensanctions/opensanctions/blob/a285915e20a1f5ea9731151b8402477c0c4a27d4/datasets/_collections/default.yml#L30-L33) and an [example](https://www.opensanctions.org/docs/entities/) that uses year but not sure if it's fine or not to store only year in `birthDate`.

3) There are two possible relevant informations in the dataset - `TOrgutAdi` which is the terrorist organization such as `DKP/BÖG` and `TKategoriAdi`which is the color-coded category assigned to the wanted individual based on it's threat level. Is this indeed relevant? If yes, what would be the appropriate metadata fields for them? 

I also bumped into [Turkey’s terror list: An attack on lawyers and human rights - Lawyers for Lawyers](https://lawyersforlawyers.org/en/turkeys-terror-list-an-attack-on-lawyers-and-human-rights/) while writing the metadata file.

### CODE REVIEW

File: datasets/tr/wanted/tr_wanted.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: tr_wanted
+title: "Turkey Wanted Terrorists"
```
Comment (jbothma) [inline datasets/tr/wanted/tr_wanted.yml:2]:
```suggestion
title: "Turkey Ministry Of Interior Terrorist Wanted List"
```

**Review by jbothma — COMMENTED** [review]
Basically perfect, thank you!

**Comment by fjuniorr** [comment]
This is the first time that I've created the entities `Organization` and `UnknownLink`, so not sure they are correct.

I also changed Turkey to Türkiye since the second form is used in the [open sanction documentation](https://www.opensanctions.org/reference/#:~:text=tr-,T%C3%BCrkiye,-tt). From [Wikipedia](https://en.wikipedia.org/wiki/Turkey):

> In December 2021, President [Recep Tayyip Erdoğan](https://en.wikipedia.org/wiki/Recep_Tayyip_Erdo%C4%9Fan) issued a [circular](https://en.wikipedia.org/wiki/Government_circular), calling for exports to be labeled "Made in Türkiye".[[53]](https://en.wikipedia.org/wiki/Turkey#cite_note-Genelge-2021/24-54) The circular also stated that in relation to other governmental communications, the "necessary sensitivity will be shown on the use of the phrase 'Türkiye' instead of phrases such as 'Turkey' (in English)".[[53]](https://en.wikipedia.org/wiki/Turkey#cite_note-Genelge-2021/24-54)[[54]](https://en.wikipedia.org/wiki/Turkey#cite_note-55) The reason given was that Türkiye "represents and expresses the culture, civilization, and values of the Turkish nation in the best way".[[53]](https://en.wikipedia.org/wiki/Turkey#cite_note-Genelge-2021/24-54) In May 2022, the Turkish government requested the [United Nations](https://en.wikipedia.org/wiki/United_Nations) and other international organizations to use Türkiye officially in English, which the UN immediately agreed to do.[[55]](https://en.wikipedia.org/wiki/Turkey#cite_note-56)[[56]](https://en.wikipedia.org/wiki/Turkey#cite_note-57)[[57]](https://en.wikipedia.org/wiki/Turkey#cite_note-58)

File: datasets/tr/wanted/tr_wanted.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  
+  Each category corresponds to a different reward amount for information that leads to the capture of these wanted individuals.
+
+  **Note**: There are concerns about the list lacking a proper legal foundation due to the absence of a legal provision that regulates who can be included, 
+  how individuals can be removed, 
+  and how authorities can establish or manage the list.
+
+  * [Turkey’s terror list: An attack on lawyers and human rights](https://lawyersforlawyers.org/en/turkeys-terror-list-an-attack-on-lawyers-and-human-rights/)
+publisher:
+  name: Ministry of Interior
```
Comment (jbothma) [inline datasets/tr/wanted/tr_wanted.yml:24]:
```suggestion
  name: Türkiye Ministry of Interior
```

File: datasets/tr/wanted/tr_wanted.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  
+  These individuals are classified into five color-coded categories, ranging from red (kırmızı) — indicating the most wanted — to blue (mavi), green (yeşil), orange (turuncu), and grey (gri). 
+  
+  Each category corresponds to a different reward amount for information that leads to the capture of these wanted individuals.
+
+  **Note**: There are concerns about the list lacking a proper legal foundation due to the absence of a legal provision that regulates who can be included, 
+  how individuals can be removed, 
+  and how authorities can establish or manage the list.
+
+  * [Turkey’s terror list: An attack on lawyers and human rights](https://lawyersforlawyers.org/en/turkeys-terror-list-an-attack-on-lawyers-and-human-rights/)
```
Comment (jbothma) [inline datasets/tr/wanted/tr_wanted.yml:22]:
```suggestion
  See [Turkey’s terror list: An attack on lawyers and human rights](https://lawyersforlawyers.org/en/turkeys-terror-list-an-attack-on-lawyers-and-human-rights/)
```

File: datasets/tr/wanted/tr_wanted.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+name: tr_wanted
+title: "Türkiye Ministry Of Interior Terrorist Wanted List"
+entry_point: crawler.py
+prefix: tr-wanted
+coverage:
+  frequency: daily
+  start: 2024-03-19
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Wanted terrorists list of Türkiye Ministry of Interior
```
Comment (jbothma) [inline datasets/tr/wanted/tr_wanted.yml:10]:
```suggestion
  List of people considered wanted for terrorism by the Türkiye Ministry of Interior
```

File: datasets/tr/wanted/tr_wanted.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
+name: tr_wanted
+title: "Türkiye Ministry Of Interior Terrorist Wanted List"
+entry_point: crawler.py
+prefix: tr-wanted
+coverage:
+  frequency: daily
+  start: 2024-03-19
```
Comment (jbothma) [inline datasets/tr/wanted/tr_wanted.yml:7]:
```suggestion
  start: 2024-04-02
```

**Pull request merged**